nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/log/LogDetailsView.java

```diff
@@ -0,0 +1,92 @@
+package org.nmrfx.processor.gui.log;
+
+import de.jensd.fx.glyphs.GlyphsDude;
+import de.jensd.fx.glyphs.fontawesome.FontAwesomeIcon;
+import javafx.event.ActionEvent;
+import javafx.geometry.HPos;
+import javafx.geometry.Insets;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Priority;
+import javafx.scene.text.Text;
+import org.nmrfx.chemistry.utilities.NvUtil;
+
+import java.awt.*;
+import java.awt.datatransfer.StringSelection;
+
+
+/**
+ * Class to display the details of a LogRecord.
+ */
+public class LogDetailsView extends GridPane {
+
+    private static final int SPACING = 5;
+    private final TextField datetime = new TextField();
+    private final TextField location = new TextField();
+    private final TextArea message = new TextArea();
+    private final Button copyButton = GlyphsDude.createIconButton(FontAwesomeIcon.COPY);
+    private String level;
+
+    public LogDetailsView() {
+        super();
```

Comment:
default constructor call to super is implicit in java, you don't need to write it.
```suggestion
```

Reply:
removed this one and the other one in LogTable